### PR TITLE
feat(cicd): update workflow for specific networks with evm

### PR DESCRIPTION
**Pull request by leisterfrancisco**
# What does this PR do?
- add workflow for hapi-evm

### CODE REVIEW

**Review by xavier506 — APPROVED** [review]
LGTM

**Review by Torresmorah — APPROVED** [review]
LGTM

File: utils/meta.mk [hunk 1]
```diff
@@ -4,13 +4,14 @@ VERSION ?= $(shell git rev-parse --short HEAD)
 
 IMAGE_NAME_WEBAPP=dashboard-webapp
 IMAGE_NAME_HAPI=dashboard-hapi
+IMAGE_NAME_HAPI_EVM=dashboard-hapi-evm
 IMAGE_NAME_HASURA=dashboard-hasura
 IMAGE_NAME_WALLET=dashboard-wallet
 
 DOCKER_REGISTRY=ghcr.io/edenia
 SUBDIRS = webapp hapi hasura wallet
```
Comment (xavier506) [inline utils/meta.mk:12]: are we missing the hapi evm in the list of subdirs? @leisterfrancisco

**Review by xavier506 — APPROVED** [review]
LGTM

**Pull request merged**